lib/Models/CkanCatalogGroup.js

```diff
@@ -548,7 +558,7 @@ function populateGroupFromResults(ckanGroup, json) {
         for (var resourceIndex = 0; resourceIndex < resources.length; ++resourceIndex) {
             var resource = resources[resourceIndex];
 
-            var groups;
+            var groups, ungrouped;
```

Comment:
This should move outside the loop for clarity.  It doesn't actually matter with the silly `var` scoping rules, but it's misleading to locate it here.  If we replaced `var` with `let`, the code would still run just fine but wouldn't work right anymore (it would create multiple "No group" groups).  I'll move it before merging this.